docs/api.md

```diff
@@ -162,6 +157,19 @@ Make an empty post to `/accounts/<username>/numbers/<SIP
 URI>/password` to generate a new password.  It is returned in the
 response as the `sip_password` parameter.
 
+Provisioning specific numbers
+=================
+
+Make an empty POST request to `/accounts/<username>/numbers/<SIP URI>/` to create that specific SIP URI. Note that:
+  * Requests to this URI *must* have the NGV-API-Key header specified - this is not an operation that ordinary users can perform
+  * The 'private_id' URL parameter is mandatory. This is assumed to be an existing private ID to associate the SIP URI with, unless the 'new_private_id' parameter is set to true, in which case this private ID will be created and the password returned.
+
+Response is the same as for a POST to `/accounts/<username>/numbers/`.
+
+Example URL:
+
+`http://ellis.example.com/accounts/Example.User%40example.com/numbers/sip:12345@example.com?private_id=private@example.com&new_private_id=true`
```

Comment:
Because I realised that it can be URL parameters or a form-encoded body (and the latter is probably more likely), I've removed the example URL and just given parameter examples in the same form as above.